src/qmflows/components/operations.py

```diff
@@ -1,26 +1,31 @@
+"""Module containing some schedule utilities."""
 
 __all__ = ['find_first_job', 'select_max', 'select_min']
 
-from noodles import schedule, find_first
+from noodles import find_first, schedule
+from scm.plams import Molecule
+
+from qmflows.packages import Result
+from qmflows.settings import Settings
```

Comment:
I'd generally recommend relative imports when within a module.

Reply:
Let's do that